csharp-design-patterns/REPRPattern/REPRPattern/BookEndpoint.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FastEndpoints;
+
+namespace REPRPattern;
+
+public class BookEndpoint : Endpoint<BookEndpointRequest, BookEndpointResponse>
+{
+    public override void Configure()
+    {
+        Post("/api/book/create");
+        AllowAnonymous();
+    }
+
+    public override async Task HandleAsync(BookEndpointRequest req, CancellationToken ct)
```

Comment:
@cjaynjoku we can improve the naming of the arguments in this method. `ct` is not contextual enough.